test/cookbooks/os_prepare/recipes/default.rb

```diff
@@ -16,7 +16,7 @@
 include_recipe('os_prepare::service')
 include_recipe('os_prepare::package')
 include_recipe('os_prepare::registry_key')
-include_recipe('os_prepare::iis_site')
+include_recipe('os_prepare::iis')
```

Comment:
This is a good move. We should add this resource to our integration tests then too